edx_proctoring/static/proctoring/js/views/proctored_exam_attempt_view.js

```diff
@@ -45,21 +45,12 @@ edx = edx || {};
     edx.instructor_dashboard.proctoring.ProctoredExamAttemptView = Backbone.View.extend({
         initialize: function() {
             this.setElement($('.student-proctored-exam-container'));
-            this.enable_exam_resume_proctoring_improvements =
-                this.$el.data('enable-exam-resume-proctoring-improvements');
-            this.enable_exam_resume_proctoring_improvements = this.enable_exam_resume_proctoring_improvements &&
-                this.enable_exam_resume_proctoring_improvements.toLowerCase() === 'true';
             this.course_id = this.$el.data('course-id');
             this.template = null;
             this.model = new edx.instructor_dashboard.proctoring.ProctoredExamAttemptModel();
 
-            if (!this.enable_exam_resume_proctoring_improvements) {
-                this.collection = new edx.instructor_dashboard.proctoring.ProctoredExamAttemptCollection();
-                this.template_url = '/static/proctoring/templates/student-proctored-exam-attempts.underscore';
-            } else {
-                this.collection = new edx.instructor_dashboard.proctoring.ProctoredExamAttemptGroupedCollection();
-                this.template_url = '/static/proctoring/templates/student-proctored-exam-attempts-grouped.underscore';
-            }
+            this.collection = new edx.instructor_dashboard.proctoring.ProctoredExamAttemptCollection();
```

Comment:
I think you rolled out the wrong logic on javascript side. The `ProctoredExamAttemptGroupedCollection` is the one you want to keep. See https://github.com/edx/edx-proctoring/pull/785/files

Reply:
I realized this is pretty confusing in the diff, I probably should have noted - I removed the old `ProctoredExamAttemptCollection`, and renamed `ProctoredExamAttemptGroupedCollection`, which now points to `StudentProctoredExamAttemptsByCourse` with the [updated grouped attempt code](https://github.com/edx/edx-proctoring/pull/820/files#diff-1674e60a26bd85080524664f75371c1b08489500da4f2a8d3fa16f8a8e2598abL990).